README.md

```diff
@@ -1,8 +1,9 @@
-# EctoJsonapi
+## Ecto Json:Api
+API |> Ecto => Json
```

Comment:
Here I was just messing around.

Is there an option that looks cool?

API |> Ecto => Json
Ecto => Json(API)
Ecto => Json:API

Otherwise I can just delete line number 2.